OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -342,7 +342,7 @@ public void Activate(Player p)
 			foreach (var defense in Info.DefenseQueues)
 				builders.Add(new BaseBuilder(this, defense, p, playerPower, playerResource));
 
-			Random = new MersenneTwister((int)p.PlayerActor.ActorID);
+			Random = new MersenneTwister(Game.CosmeticRandom.Next());
```

Comment:
This works, but I would use only Random = new MersenneTwister(); so without dependency on Game.CosmeticRandom.